default/python/AI/freeorion_tools/_freeorion_tools.py

```diff
@@ -347,3 +348,22 @@ def wrapper(*args, **kwargs):
 
         return wrapper
     return decorator
+
+
+def object_is_in_ion_storm(object_id):
+    # FIXME: This isn't working if we can not detect the ion storm
+    #        as is the case if the center of the storm is out of detection range
+    universe = fo.getUniverse()
+    universe_object = universe.getPlanet(object_id)
```

Comment:
If we want to restrict the function to only dealing with planets, then we should rename it.  But it seems better to me to change this to  
```
universe_object =  universe.getObject(object_id)
```